resources/public/api/conf/1.0/broughtForwardLosses_list.raml

```diff
@@ -0,0 +1,33 @@
+description: Retrieve all Brought Forward Losses
+is:
+  - headers.acceptJson
+  - headers.authorization
+  - errors.notFound
+  - errors.formatTaxYear
+  - errors.agentSubscription
+  - errors.clientSubscriptionToMTD
+queryParameters:
+  taxYear:
+    description: Tax Year
+    example: "2017-18"
+    required: false
+  lossType:
+    description: "Type of Loss.  Can be one of the four loss types: <b><ul> <li> self-employment  <li>self-employment-class4  <li>uk-fhl-property   <li>uk-other-property </b>"
+    example: "self-employment"
+    required: false
+responses:
+  200:
+    body:
+      application/json:
+        description: Retrieve all brought forward losses
+        #type: !include schemas/ListAllBFLoss.json
```

Comment:
Please delete the commented ones... they are redundant

Reply:
Again, these are from the boilerplate. Would you like me to remove them as part of my PR or leave them until the individual stories are worked? You're commenting on files that were created before I branched off your branch.